README.md

```diff
@@ -369,7 +346,7 @@ The default port is 1337, to use a different port set the PORT environment varia
 $ PORT=8080 parse-server --appId APPLICATION_ID --masterKey MASTER_KEY
 ```
 
-For the full list of configurable environment variables, run `parse-server --help`.
+For the full list of configurable environment variables, run `parse-server --help` or [Parse Server Configurations](http://parseplatform.org/parse-server/api/master/ParseServerOptions.html).
```

Comment:
```suggestion
For the full list of configurable environment variables, run `parse-server --help` or take a look at [Parse Server Configurations](http://parseplatform.org/parse-server/api/master/ParseServerOptions.html).
```

Reply:
I just realized the env variables aren't generated.